Sources/HTTP/FoundationClient/URI+Foundation.swift

```diff
@@ -11,8 +11,8 @@ extension URI {
         comps.scheme = scheme.isEmpty ? nil : scheme
         comps.user = userInfo?.username
         comps.password = userInfo?.info
-        comps.host = host.isEmpty ? nil : host
-        comps.port = port
+        comps.host = hostname.isEmpty ? nil : hostname
+        comps.port = port == nil ? nil : Int(port!)
```

Comment:
I'd rather use `if let` here or `port.flatMap { Int($0) }` to avoid the `!` even though it's technically save w/ `== nil`